schemas/com.vimeo/video/jsonschema/1-0-0

```diff
@@ -0,0 +1,90 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a Vimeo video.",
+  "self": {
+    "vendor": "com.vimeo",
+    "name": "video",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "videoId": {
+      "type": "number",
+      "description": "The Vimeo ID of the video.",
+      "minimum": 0,
+      "maximum": 9223372036854776000
+    },
+    "videoTitle": {
```

Comment:
Nit: this could also be stored in the `label` property of the `player` media schema, but either way is fine.

Reply:
Just to close this one off - I think we settled on `label` being more of an identifier for the player, rather than a specific video title.